<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCjyxPlotsModuleWidget</class>
 <widget class="qCjyxWidget" name="qCjyxPlotsModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Plots</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qCjyxPlotsModule.qrc">
    <normaloff>:/Icons/Plots.png</normaloff>:/Icons/Plots.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="PlotsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="charts">
      <attribute name="title">
       <string>Charts</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="PlotChartLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>Chart:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="qDMMLNodeComboBox" name="PlotChartNodeSelector">
           <property name="nodeTypes">
            <stringlist>
             <string>vtkDMMLPlotChartNode</string>
            </stringlist>
           </property>
           <property name="renameEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ShowChartButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Allow plot editing</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../Base/QTGUI/Resources/qCjyxBaseQTGUI.qrc">
             <normaloff>:/Icons/Medium/CjyxVisible.png</normaloff>:/Icons/Medium/CjyxVisible.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="qDMMLPlotChartPropertiesWidget" name="PlotChartPropertiesWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="series">
      <attribute name="title">
       <string>Series</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="PlotDataSeriesLabel">
         <property name="text">
          <string>Data series:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="qDMMLNodeComboBox" name="PlotSeriesNodeSelector">
         <property name="nodeTypes">
          <stringlist>
           <string>vtkDMMLPlotSeriesNode</string>
          </stringlist>
         </property>
         <property name="renameEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="qDMMLPlotSeriesPropertiesWidget" name="PlotSeriesPropertiesWidget"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="ctkPushButton" name="copyPlotSeriesNodePushButton">
         <property name="text">
          <string>Clone data series</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="BottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qDMMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qDMMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLWidget</class>
   <extends>QWidget</extends>
   <header>qDMMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCjyxWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLPlotChartPropertiesWidget</class>
   <extends>qDMMLWidget</extends>
   <header>qDMMLPlotChartPropertiesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLPlotSeriesPropertiesWidget</class>
   <extends>qDMMLWidget</extends>
   <header>qDMMLPlotSeriesPropertiesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qCjyxPlotsModule.qrc"/>
  <include location="../../../../../Base/QTGUI/Resources/qCjyxBaseQTGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>PlotChartNodeSelector</sender>
   <signal>currentNodeChanged(vtkDMMLNode*)</signal>
   <receiver>PlotChartPropertiesWidget</receiver>
   <slot>setDMMLPlotChartNode(vtkDMMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxPlotsModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>PlotChartNodeSelector</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxPlotsModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>PlotSeriesPropertiesWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>708</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxPlotsModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>PlotSeriesNodeSelector</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlotSeriesNodeSelector</sender>
   <signal>currentNodeChanged(vtkDMMLNode*)</signal>
   <receiver>PlotSeriesPropertiesWidget</receiver>
   <slot>setDMMLPlotSeriesNode(vtkDMMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>597</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PlotChartNodeSelector</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>ShowChartButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxPlotsModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>PlotChartPropertiesWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
